spec/features/adventure_spec.rb

```diff
@@ -116,6 +116,16 @@
         expect(page).to have_content("Add scene")
       end
 
+      it "displays age verification" do
+        visit new_adventure_path
+
+        expect(page).to have_content("Add Age Verification?")
+        expect(page).to have_content("If checked, others must verify their age to engage this Story.")
+
+        find(:css, "#adventure_has_age_limit").set(true)
+        expect(page).to have_content("Age Limit")
+      end
+
```

Comment:
would be good to get some specs in place testing that the age gate itself works, similar to what exists for the password check: https://github.com/vigetlabs/storyboard/blob/dcca328d61886086fe73cb8ebe2c5acb64e8fc44/spec/features/adventure_spec.rb#L172-L198